contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -242,7 +242,7 @@ def test_aaf_read(self):
 
         self.assertEqual(len(timeline.audio_tracks()), 2)
 
-        clips = list(video_track.each_clip())
+        clips = list(video_track.find_clips())
```

Comment:
I suspect the `list(...)` wrapper here is redundant now that `find_clips()` returns a list.

Reply:
Good catch, I also removed a couple of other redundant ```list()``` calls.